packages/cli/index.ts

```diff
@@ -90,36 +90,20 @@ const main = defineCommand({
           },
         },
         run: async ({ args }) => {
-          const paths = args._.length
+          const paths = args._.length > 0
             ? await glob(args._, {
                 expandDirectories: false,
                 onlyDirectories: true,
                 absolute: true,
```

Comment:
Today I found out that my script isn't working. Was it because I replaced fast-glob here with an absolute path?

<img width="663" alt="image" src="https://github.com/user-attachments/assets/5fb3103e-27c7-49c1-86cc-a4cebcf1c3ef" />

Should I change it like this?

<img width="559" alt="image" src="https://github.com/user-attachments/assets/d09c7b62-223a-44f6-8d81-3b7c1e817dd0" />